ui_test/POM_hiway_asg/pages/timesheet.py

```diff
@@ -0,0 +1,156 @@
+import time
+from POM_hiway_asg.pages.login_using_google import LoginWithGoogle as LWG
+from POM_hiway_asg.pages.hiway_homepage import HiwayHomepageOptions
+from POM_hiway_asg.pages.constant_var import *
+from selenium.webdriver.common.keys import Keys
+from selenium.common.exceptions import NoSuchElementException
+from datetime import date, timedelta
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') standard import "from datetime import date, timedelta" should be placed before "from POM_hiway_asg.pages.login_using_google import LoginWithGoogle as LWG" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=Pylint%3AC0411)